prestoadmin/util/presto_config.py

```diff
@@ -66,12 +66,17 @@ def coordinator_config():
         try:
             data = StringIO()
             with settings(host_string='%s@%s' % (env.user, config_host)):
-                get(config_path, data, use_sudo=True)
+                with hide('stderr', 'stdout'):
+                    temp_dir = sudo('mktemp -d /tmp/prestoadmin.XXXXXXXXXXXXXX')
```

Comment:
does this need sudo?  At some point we're going to need to work on purging unnecessary sudo calls.  If it does need sudo, maybe just generate a random string here, and pass it in as temp_dir